.readthedocs.yaml

```diff
@@ -9,6 +9,10 @@ sphinx:
 formats: [htmlzip]
 
 python:
-   version: "3.8"
    install:
    - requirements: docs/.sphinx/requirements.txt
+
+build:
+   os: ubuntu-20.04
```

Comment:
This won't affect me if I build locally with Ubuntu22 will  it?

Reply:
No, this is just what the read the docs server uses.